### DISCUSSION TITLE
Radio mediums

### DISCUSSION THREAD

**Discussion by agates** [Podcastindex-org/podcast-namespace#567] [Enhancement Proposal]
_EDIT: This needs to be rewritten as of 2023-09-01.  This was merely the original idea, but I want to add additional considerations based off recent PC20 discussion._

Thinking about what I briefly discussed with @samsethi on Podnews Weekly Review, I'd like to suggest a `radio` (and `radioL` playlist) medium.

This medium would be dedicated to 24/7 live streams that don't expect regular updates in their `<podcast:liveItem>` metadata.  For example, the No Agenda Live stream would have a medium of "radio" and its `liveItem` title would only be "No Agenda Stream Live".  Individual shows would remain in their own feeds.  Similarly, 100% Retro could be re-made as a radio medium.

The feeds would not have regular items as they would be focused on the live streams alone.  Podping notifications would not be expected as the liveItems would never or rarely update.

The real life analog to this being the existing radio stations and internet radio streams where users tune in any time of day and don't necessarily know or care about exactly what's playing before they turn it on.

**Comment by dhk2** [comment]
Updating the liveitem with the remoteitem for the currently broadcasting podcast seems like it would be useful.

**Reply by agates** [reply]
I've been throwing the idea around in my head to have some kind of item-to-item references for various reasons for some time now.  I think it would fall under the same concept.  Just haven't really went down that path to give everyone a chance to catch up.  Maybe this is the right opportunity.

I can already see people asking about how to utilize the value tag and it feels like it's just going to be a mess, although perhaps the delay of Podping wouldn't be so bad for shows that are like an hour+ long.

**Comment by samsethi** [comment]
Hey Alecks 

Thanks for this.  As you know we have already added Radio as a "Medium" parameter.  

<img width="1431" alt="Fullscreen_30_08_2023__14_42" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/698662d7-b7b9-4fce-8786-bf4bd8fc9d1a">

We just need to change the start/end time on the LiveItem to continuous.  The initial streaming sats would still be set by the content owner but that can be changed by the user i.e Value4Value.  

<img width="1149" alt="Podfans___Dashboard" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/2e3e18e3-bb57-4c22-84af-a6c7bb4db332">

Is there a good example of a potential radio station we could work with to test this?

**Reply by agates** [reply]
I don't like the idea of putting a non-ISO8601 value into the start/end fields.  Would a new `continuous=[true|false]` attribute work?  We'd have to make start optional, but this feels like it would convey the intent in the most obvious manner.

**Reply by agates** [reply]
I think 100% Retro could be adapted fairly easily if @daveajones was up for it.  And the No Agenda Stream seems like a no-brainer.  Otherwise I have no insight into the industry.

**Reply by daveajones** [reply]
I can totally change it to this new style. The existing feed was a proof of concept anyway.

Actually I could make a second feed for it also so that both styles are active.

**Comment by samsethi** [comment]
Did this additional attribute for continuous ever get implemented?

**Reply by agates** [reply]
I don't think we ever talked about it outside of this discussion.
